Makefile

```diff
@@ -1,6 +1,7 @@
 BUILD_DIR=$(shell pwd)/build
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
 CERT_GEN=$(shell base64 -w 0 scripts/cert-gen.sh)
+VERSION ?= $(shell cat VERSION)
```

Comment:
@hferentschik only reason I can think of to maintain version info in single file and then use it across. In our case we can have it in the Makefile itself. I will remove that version file and directly add version info to `VERSION` variable.